client/state/comments/test/actions.js

```diff
@@ -0,0 +1,358 @@
26T18:11:19+00:00"}},"753":{"children":[],"parentId":714,"data":{"date":"2016-01-26T19:44:38+00:00"}},"754":{"children":[],"parentId":735,"data":{"date":"2016-01-26T19:46:54+00:00"}},"755":{"children":[],"parentId":718,"data":{"date":"2016-01-26T21:21:20+00:00"}},"756":{"children":[],"parentId":740,"data":{"date":"2016-01-27T01:01:44+00:00"}},"757":{"children":[759],"parentId":null,"data":{"date":"2016-01-27T02:06:24+00:00"}},"759":{"children":[],"parentId":757,"data":{"date":"2016-01-27T09:09:37+00:00"}},"760":{"children":[769],"parentId":null,"data":{"date":"2016-01-27T11:25:21+00:00"}},"761":{"children":[770],"parentId":null,"data":{"date":"2016-01-27T14:56:47+00:00"}},"762":{"children":[771],"parentId":null,"data":{"date":"2016-01-27T15:47:46+00:00"}},"763":{"children":[772],"parentId":null,"data":{"date":"2016-01-27T21:47:52+00:00"}},"764":{"children":[773],"parentId":null,"data":{"date":"2016-01-27T22:18:44+00:00"}},"765":{"children":[774],"parentId":null,"data":{"date":"2016-01-28T00:04:06+00:00"}},"766":{"children":[],"parentId":null,"data":{"date":"2016-01-28T05:48:13+00:00"}},"768":{"children":[775],"parentId":null,"data":{"date":"2016-01-29T02:14:32+00:00"}},"769":{"children":[778],"parentId":760,"data":{"date":"2016-01-29T11:06:29+00:00"}},"770":{"children":[],"parentId":761,"data":{"date":"2016-01-29T11:09:00+00:00"}},"771":{"children":[],"parentId":762,"data":{"date":"2016-01-29T11:10:01+00:00"}},"772":{"children":[],"parentId":763,"data":{"date":"2016-01-29T11:10:23+00:00"}},"773":{"children":[],"parentId":764,"data":{"date":"2016-01-29T11:10:38+00:00"}},"774":{"children":[],"parentId":765,"data":{"date":"2016-01-29T11:11:38+00:00"}},"775":{"children":[],"parentId":768,"data":{"date":"2016-01-29T11:12:16+00:00"}},"776":{"children":[],"parentId":null,"data":{"date":"2016-01-29T12:40:19+00:00"}},"777":{"children":[781],"parentId":null,"data":{"date":"2016-01-29T16:58:22+00:00"}},"778":{"children":[],"parentId":769,"data":{"date":"2016-01-29T19:26:31+00:00"}},"781":{"children":[],"parentId":777,"data":{"date":"2016-01-30T21:32:54+00:00"}},"782":{"children":[788],"parentId":null,"data":{"date":"2016-01-31T05:34:55+00:00"}},"783":{"children":[787],"parentId":null,"data":{"date":"2016-01-31T09:29:48+00:00"}},"785":{"children":[790],"parentId":null,"data":{"date":"2016-01-31T15:09:30+00:00"}},"786":{"children":[],"parentId":null,"data":{"date":"2016-01-31T16:10:30+00:00"}},"787":{"children":[],"parentId":783,"data":{"date":"2016-02-01T17:57:47+00:00"}},"788":{"children":[],"parentId":782,"data":{"date":"2016-02-01T17:58:04+00:00"}},"790":{"children":[],"parentId":785,"data":{"date":"2016-02-01T17:59:14+00:00"}},"791":{"children":[792],"parentId":null,"data":{"date":"2016-02-02T20:10:54+00:00"}},"792":{"children":[],"parentId":791,"data":{"date":"2016-02-03T10:16:35+00:00"}},"793":{"children":[794],"parentId":null,"data":{"date":"2016-02-04T01:07:58+00:00"}},"794":{"children":[],"parentId":793,"data":{"date":"2016-02-05T11:15:36+00:00"}}};
```

Comment:
Sorry I haven't wrote it earlier here, I've already addressed it a while ago by making that data [much smaller](https://github.com/Automattic/wp-calypso/pull/3143/files#diff-7683b0e3a8968ec7d9b075cd827dc5daR205).
However, I'll be looking more into how to make the tests more simpler.